test/jdk/java/foreign/capturecallstate/TestCaptureCallState.java

```diff
@@ -61,16 +61,12 @@ public class TestCaptureCallState extends NativeTestHelper {
         }
     }
 
-    private record SaveValuesCase(String nativeTarget, FunctionDescriptor nativeDesc, boolean trivial, String threadLocalName, Consumer<Object> resultCheck) {}
```

Comment:
For the changes in this file, I've just reverting things to how the were before: https://github.com/openjdk/panama-foreign/pull/848